services/celery/celery.py

```diff
@@ -50,20 +57,25 @@ def open_order_on_dydx():
             order_manager.store_data(dydx_order_details, "dydx_orders")
             order_id = dydx_order_details["id"]
             open_position = True
+            print("SELL order is  placed")
 
     else:
         print("SELL order is already placed")
 
 
 @app.task(name="close_order_on_dydx", default_retry_delay=4 * 60)
-def close_order_on_dydx():
+def close_order_on_dydx(trigger_price_test=None):
     global open_position
     global order_id
     if open_position is True:
         order_manager_obj = OrderManager()
         order_params = order_manager_obj.order_params()
         # TODO: write a formula to calculate  the trigger_price
-        trigger_price = 1300
+        trigger_price = ""
+        if trigger_price_test is None:
+            trigger_price = 1100
```

Comment:
1100 give this a variable because it's just a random value that we're using to run our tasks until we get the proper formula. 
lol, infact, you can call this **trigger_price_test**

Reply:
no we can't change trigger_price =1100 to trigger_price_test because see the below if conditions .